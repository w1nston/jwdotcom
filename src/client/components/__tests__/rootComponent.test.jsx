import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import RootComponent from '../rootComponent';

describe('<RootComponent/>', () => {
  it('renders', () => {
    const Router = jest.fn();
    const component = shallow(<RootComponent Router={Router} />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
