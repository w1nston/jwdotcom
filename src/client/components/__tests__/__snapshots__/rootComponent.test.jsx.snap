exports[`<RootComponent/> renders 1`] = `
<mockConstructor>
  <div>
    <nav
      className="navigation-container">
      <img
        className="navigation-container__logo"
        src="/assets/images/jwlogo.svg" />
      <ul
        className="navigation-container__items">
        <li
          className="navigation-container__item">
          <Link
            replace={false}
            to="/">
            Home
          </Link>
        </li>
        <li
          className="navigation-container__item">
          <Link
            replace={false}
            to="/blog">
            Blog
          </Link>
        </li>
      </ul>
    </nav>
    <Route
      component={[Function]}
      exact={true}
      path="/" />
    <Route
      component={[Function]}
      path="/blog" />
  </div>
</mockConstructor>
`;
